src/docs/project_goals.md

```diff
@@ -0,0 +1,48 @@
+
+## Desktop & Browser, Local & Remote (Cloud)
+
+Tools and IDEs built on Theia can run as desktop applications or in the browser. The backend of Theia can run locally (for the desktop case) or as a remote service in the cloud. Both variants currently have an equal priority.
+
+## Platform for IDEs and Tools
+Theia is targeted at building a variety of custom tools. Prominent examples of Theia-based products include IDE's, but Theia also explicitly targets tools that do not focus on textual input/editors or software development. Thus, Theia's scope also included diagram editors or form-based UI. The Theia platform hence covers the typical application scenarios of the Eclipse rich-client platform (Eclipse RCP). As a consequence, the platform aims at making as few assumptions as possible about what adopters might want to build with Theia. It explicitly allows deviation from the standard workbench layout and the removal of all default features.
+
+## Basic Workbench Frame
+
+Theia provides a workbench frame, i.e., a window management system permitting the display of views, editors, and menus and makes available tools and interactions allowing the user to modify of the window layout, trigger commands, and use key bindings and other concepts known from a desktop tool such as drag and drop.
+
+## Reusable Common Tool Features
+
+Theia optionally provides common tool features as components to be reused by adopters. Common features are generic so that may be reused in several tools. Examples for these common tool features include a file explorer, Git support and a code editor.
+
+## Extensibility and Adaptability
+
+Theia is an extensible and adaptable framework. Extensibility in this context means that you can easily add new features to a product built on the Theia platform (including UI and backend functionality). These new features can either be provided by the Theia project itself (common features), by other projects (e.g. Eclipse GLSP) or be custom features that are developed by an adopter. Adaptable in this context means that the workbench and all common features that are provided by the Theia project can be customized and adapted to project-specific needs. This includes changing or removing existing features and adjusting the look and feel.
+
+## VS Code compatibility
+
+Theia provides the ability to host VS Code extensions. This allows adopters to benefit from features that are provided as VS Code extensions and make them part of their tool offering. It also allows end users of Theia-based tools to install additional features, if the adopter providing a tool allows users to do so. When applicable, the default Theia UX aligns with the VS Code UI, although adopters can deviate from this.
+
+## Use standards and don’t reinvent the wheel
```

Comment:
minor: to align with other titles (capitalized first letters), we might want to find a shorter title for this section.

Reply:
capitalized it, if you have an idea for a better title, please suggest. This was actually under discussion before and we ended upt with this long title to make 100% sure what we mean (it is actually kind of two things that are heavily related)

Reply:
Keeping it like you have it but capitalized seems good to me :+1: